modules/trusted-firmware-m/tfm_boards/services/include/tfm_platform_user_memory_ranges.h

```diff
@@ -76,4 +76,26 @@ static const struct tfm_read_service_range ranges[] = {
 #endif
 };
 
-#endif /* TFM_READ_RANGES_H__ */
+#if defined(NRF_RRAMC_S)
+/**
+ * 0x518 is the offset for the low power configuration, currently not in MDK,
+ * the first two bits control the lower power mode of RRAMC.
+ */
+#define NRF_RRAMC_LOWPOWER_CONFIG_ADDR (NRF_RRAMC_S_BASE + 0x518)
+
+/* Values: 0x0 = Power down mode, 0x1 = Standby mode */
+static const uint32_t rramc_lowpower_config_allowed[] = {0x0, 0x1};
+#endif /* NRF_RRAMC_S */
+
+static const struct tfm_write32_service_address tfm_write32_service_addresses[] = {
+	/* This is a dummy value because this table cannot be empty */
+	{.addr = 0xFFFFFFFF, .mask = 0x0, .allowed_values = NULL, .allowed_values_array_size = 0},
```

Comment:
Why not have this only `#ifndef NRF_RRAMC_S`?

Reply:
Do you refer to the dummy value here?

Reply:
Yes.

Reply:
Now doing this it makes sense, but afterwards I expect that this list will be expanded with more entries and having an ifndef for all the other entries doesn't seem very good to me.

Reply:
Of course it can then be modified appropriately later on. I would just leave out those entries that are not needed by principle (and heh, saves a few bytes). But I'm not familiar with this code anyway. Just thinking out loud.

Reply:
So the thing is that we need at least one entry, otherwise the build will fail all together. I added an  if/else to avoid having the dummy value when we have rramc defined and we can decide what do when we have more entries later. This will save some bytes when RRAMC is enable but it will still add the dummy value when the RRAMC is disabled. Saving these bytes requires allowing to enable/disable specific platform services which is not supported at the moment and it is not in scope for this PR.